knife/knife.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.email = "adam@chef.io" # These seem a bit out of date, and this address probably doesn't go anywhere anymore?
   s.homepage = "https://www.chef.io"
 
-  s.required_ruby_version = ">= 2.7.0"
+  s.required_ruby_version = ">= 3.0.0"
```

Comment:
@johnmccrae knife is already at >= 2.7.0, so this change should not be needed right? This is breaking change

Reply:
We should not be bumping this on Chef 17

Reply:
That's a good point. I was afraid that if we kept it at 2.7 there might be test problems. I'll roll it back.